### PR TITLE
Update release docs

### LINKED ISSUE #18743: ML.Net Extension Uninstall (Documentation) is not sufficient
VS2019 will install the `ML.Net Model Builder (Preview)` extension. The uninstall instructions on the web page tell me to "Manage Extensions", use the "Uninstall" button, and "follow the instructions".

**Problem**
The "Uninstall" button will open the Visual Studio Installer. However, there are no _instructions to follow_. Well, I was looking through my installed workloads (Azure as the first suspect), but couldn't find it at first. Searching all other sections, I finally found it under both of the following sections:

- .NET Core cross-platform development
- Individual components

Unticking both (!) will show -277 MB and uninstall the extension.

**Suggestion**
The uninstall documentation should explain how to proceed with Visual Studio Installer, because _there are no instructions to follow_. Maybe add a screenshot.
Clarify: Be sure to discover all occurrences, not just one.

Even better: change the uninstall button behavior (maybe some CLI arguments into the VS Installer) to directly make the necessary modifications in the Visual Studio Installer. Then users just have to confirm the changes to make the uninstall happen. This would be the best user experience, but eventually out of scope here :)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5692cfb5-52e1-b6b9-2b87-7ea0decc2882
* Version Independent ID: 2da762a6-5bc4-a8d1-afae-f1a8243eee33
* Content: [How to install Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder)
* Content Source: [docs/machine-learning/how-to-guides/install-model-builder.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/how-to-guides/install-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### LINKED ISSUE #22832: Select scenario screenshot is outdated
The "Select a scenario" screenshot is outdated since now there are other available scenarios:

- Text classification
- Value prediction
- Image classification
- Recommendation
- Object detection

However, the article speaks about "Issue Classification".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f7fff31d-9b11-e1f1-f4c1-d798a7d9b564
* Version Independent ID: 8c94de21-2cc2-fb08-314d-6a62af1f0421
* Content: [Tutorial: Classify health violations with Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/health-violation-classification-model-builder)
* Content Source: [docs/machine-learning/tutorials/health-violation-classification-model-builder.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/health-violation-classification-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### LINKED ISSUE #25921: Rename instances of Text Classification to Data Classification
Rename instances of Text Classification to Data Classification to align with changes in 

dotnet/machinelearning-modelbuilder#1710

### LINKED ISSUE #25922: Clarify data types for classification scenarios (binary / multiclass) in Model Builder
For classification scenarios, depending on the label column's data type, a model is trained using binary or multiclass classification.

Bool -> Binary
String -> Multiclass

Users still have control over this in the advanced data options tab.

The best place for this change might be in the [Model Builder data loading how-to guide](https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/load-data-model-builder).

See dotnet/machinelearning-modelbuilder#1211 for more details.

### LINKED ISSUE #26089: Model Builder install instructions
With VS2022, there's no longer a need to check the experimental flag.

### LINKED ISSUE #26179: Update Model Builder samples to Add Machine Learning Model
The option to train an ML model with Model Builder is no longer labeled "Add Machine Learning" it's now "Add Machine Learning Model". Update samples to reflect this change.

image

### PR DESCRIPTION
**Pull request by luisquintanilla**
Update docs based on Sept 2021 Model Builder release

- Updated Model Builder install docs to include VS2022
- Add note on binary classification based on label data type
- Updates scenario screen screenshots
- Update instructions on how to add machine learning model
- Update instances of text classification to data classification

Fixes #26089
Fixes #25922 
Fixes #18743
Fixes #26179
Fixes #22832
Fixes #25921 

## Preview Links

File Type | File Name | Published Url
-- | -- | --
Content | docs/machine-learning/how-to-guides/install-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder?branch=pr-en-us-26148
Content | docs/machine-learning/how-to-guides/load-data-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-model-builder?branch=pr-en-us-26148
Content | docs/machine-learning/tutorials/health-violation-classification-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/health-violation-classification-model-builder?branch=pr-en-us-26148
Content | docs/machine-learning/tutorials/object-detection-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder?branch=pr-en-us-26148
Content | docs/machine-learning/tutorials/predict-prices-with-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/predict-prices-with-model-builder?branch=pr-en-us-26148
Content | docs/machine-learning/tutorials/sentiment-analysis-model-builder.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis-model-builder?branch=pr-en-us-26148

### CODE REVIEW

**Comment by luisquintanilla** [comment]
@jwood803 can I get a review when you get a change. Thanks.

**Review by jwood803 — APPROVED** [review]
LGTM!

**Review by briacht — APPROVED** [review]

File: docs/machine-learning/how-to-guides/install-model-builder.md [hunk 1]
```diff
@@ -17,54 +17,64 @@ Learn how to install ML.NET Model Builder to add machine learning to your .NET a
...
+    :::image type="content" source="media/install-model-builder/disable-model-builder-visual-studio-2022.png" alt-text="Disable Model Builder Visual Studio 2022" lightbox="media/install-model-builder/disable-model-builder-visual-studio-2022.png":::
 
-    ![VS2019 search and uninstall Model Builder extension in extensions manager dialog](./media/install-model-builder/vs2019-uninstall-model-builder.png)
+---
 
-1. Follow the prompts to complete the uninstallation.
+## Upgrade Model Builder
 
-## Upgrade
+Model Builder automatically updates when there's a new version.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/install-model-builder.md:78]: It should, but it's not consistent. If you want to make sure you're on the latest version, then a manual update via extension manager is the way to go. 

@JakeRadMSFT thoughts on recommended update instructions?

File: docs/machine-learning/tutorials/object-detection-model-builder.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ In Visual Studio, create a **C# .NET Core console application** called *StopSign
 
 ## Create an `mbconfig` file
 
-1. In **Solution Explorer**, right-click the *StopSignDetection* project, and select **Add** > **Machine Learning** to open the Model Builder UI.
+1. In **Solution Explorer**, right-click the *StopSignDetection* project, and select **Add** > **Machine Learning Model...** to open the Model Builder UI.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/object-detection-model-builder.md:124]: Try a hard refresh. I've noticed similar issues on the preview, but the live site looks okay. I've made no changes to the images here so I would expect them to still show up.

https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder

File: docs/machine-learning/how-to-guides/install-model-builder.md [hunk 3]
```diff
@@ -17,54 +17,64 @@ Learn how to install ML.NET Model Builder to add machine learning to your .NET a
...
 
-The upgrade process is similar to the installation process. Either download the latest version from Visual Studio Marketplace or use the Extensions Manager in Visual Studio.
+However, if you'd prefer to manually install the latest version, either download it version from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=MLNET.07) or use the Extensions Manager in Visual Studio. See [how to update a Visual Studio extension](/visualstudio/extensibility/how-to-update-a-visual-studio-extension) for more information.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/install-model-builder.md:80]: Resolved

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 4]
```diff
@@ -18,22 +18,26 @@ Model Builder only accepts TSV, CSV, and TXT files with comma, tab, and semi-col
...
 
 1. In the data step of Model Builder, select **File** as the data source type.
-2. Select the **Browse** button next to the text box, and use File Explorer to browse and select the data file.
-3. Choose a category in the **Column to Predict (Label)** dropdown.
-4. Update the data in the **Advanced data options** link to set column settings or to update the data formatting.
+1. Select the **Browse** button next to the text box, and use File Explorer to browse and select the data file.
+1. Choose a category in the **Column to Predict (Label)** dropdown.
+
+    > ![NOTE]
+    > **Data classification scenarios**: If the data type of your label column (the value in the "Columns to Predict (Label)" dropdown) is set to Boolean (True/False), a binary classification algorithm is used in your model training pipeline. Otherwise, a multiclass classification trainer is used. Use **Advanced data options** to modify the data type for your label column and inform Model Builder which type of trainer it should use for your data.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/load-data-model-builder.md:38]: Resolved

File: docs/machine-learning/tutorials/health-violation-classification-model-builder.md [hunk 5]
```diff
@@ -64,16 +64,16 @@ The `label` is the column you want to predict. When performing a classification
 
 When first adding Model Builder to the solution it will prompt you to create an `mbconfig` file. The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
 
-1. In **Solution Explorer**, right-click the *RestaurantViolations* project, and select **Add** > **Machine Learning**.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/health-violation-classification-model-builder.md:67]: Resolved

File: docs/machine-learning/tutorials/health-violation-classification-model-builder.md [hunk 6]
```diff
@@ -64,16 +64,16 @@ The `label` is the column you want to predict. When performing a classification
...
 ## Choose a scenario
 
-![Model Builder wizard in Visual Studio](../media/model-builder-screen.png)
+![Model Builder wizard in Visual Studio](../media/model-builder-scenarios.png)
 
-To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is *Text Classification*.
+To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is *Data Classification*.
 
-1. For this sample, the scenario is multiclass classification. In the *Scenario* step of Model Builder, select the **Issue Classification** scenario.
+1. For this sample, the scenario is multiclass classification. In the *Scenario* step of Model Builder, select the **Data Classification** scenario.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/health-violation-classification-model-builder.md:76]: Yeah. That sounds better to keep it consistent with how we think about tasks + algos & scenarios.

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 7]
```diff
@@ -53,6 +57,10 @@ To load data from a SQL Server database file into Module Builder:
     1. Select **OK**
 1. Choose the dataset name from the **Table Name** dropdown.
 1. From the **Column to Predict (Label)** dropdown, choose the data category on which you want to make a prediction.
+
+    > ![NOTE]
+    > **Data classification scenarios**: If the data type of your label column (the value in the "Columns to Predict (Label)" dropdown) is set to Boolean (True/False), a binary classification algorithm is used in your model training pipeline. Otherwise, a multiclass classification trainer is used. Use **Advanced data options** to modify the data type for your label column and inform Model Builder which type of trainer it should use for your data.
+
 1. Update the data in the **Advanced data options** link to set column settings or to update the data formatting.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/load-data-model-builder.md:64]: Resolved

File: docs/machine-learning/tutorials/health-violation-classification-model-builder.md [hunk 8]
```diff
@@ -64,16 +64,16 @@ The `label` is the column you want to predict. When performing a classification
...
+1. In **Solution Explorer**, right-click the *RestaurantViolations* project, and select **Add** > **Machine Learning Model...**.
 1. Name the `mbconfig` project **RestaurantViolationsPrediction**, and click the **Add** button.
 
 ## Choose a scenario
 
-![Model Builder wizard in Visual Studio](../media/model-builder-screen.png)
+![Model Builder wizard in Visual Studio](../media/model-builder-scenarios.png)
 
-To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is *Text Classification*.
+To train your model, select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is *Data Classification*.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/health-violation-classification-model-builder.md:74]: Resolved

File: docs/machine-learning/tutorials/object-detection-model-builder.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/object-detection-model-builder.md:124]: Resolved

File: docs/machine-learning/how-to-guides/install-model-builder.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JakeRadMSFT) [reply docs/machine-learning/how-to-guides/install-model-builder.md:78]: I think it's fine - we update "Automatically" in the same way as all other extensions. We could open up a bug against VS if there isn't one already.

[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/install-model-builder.md:78]: Sounds good. Thanks.

**Comment by adegeo** [comment]
@luisquintanilla Hey there. Please double check that the checks are passing before you merge in. 😁

**Pull request merged**